model-optimizer/mo/moc_frontend/extractor.py

```diff
@@ -29,8 +29,10 @@ def decode_name_with_port(input_model: InputModel, node_name: str):
 
     node = input_model.get_place_by_operation_name(node_name)
     if node:
-        found_node_names.append('Operation:' + node_name)
-        found_nodes.append(node)
+        node = node.get_output_port(node_name)
```

Comment:
@slyalin Can you please have a look on this? "Operation places don't have `is_equal_data`". Is it a generic frontend requirement so that MO can rely on this?

Reply:
Looking at the code I don't understand the problem we are trying to solve here. Could one explain?

Here is a general thought about it. Place has many methods that can be conditionally applied depending on place kind. Because it is a fat interface. If some method can be interpreted correctly for some place regardless of its type, it should. For operation places which has a single output port, we can associate 'data' with this output tensor. So if two places have the same tensor associated with them, then is_equal_data should return true, not fail. If there is no unambiguous data association for one of the places, then it should return false. Fail-or-false decision should come from real practical applications. I think false is better.

The original description at https://github.com/openvinotoolkit/openvino/blob/master/ngraph/frontend/frontend_manager/include/frontend_manager/place.hpp#L297-L303 doesn't involve an op place in the equation. If we find it useful to include op as I have described above (as unique output data for op), for example, because code becomes cleaner and shorter, let's do it.

Reply:
In this case my suggestion would be to change `node.get_output_port(node_name)` to `node.get_output_port()`?
Argument of 'get_output_port(name)' is a name of port group, there is no such guarantee in PaddlePaddle about this.
Also if node has more than one output (and there is no API) - FrontEnd can raise exception (PDPD, TF), so it looks good to wrap it with try/catch

Reply:
Ho 'legacy onnx' worked for such models? If it worked fine, maybe in this case we should just remove completely this block from line 30 to 33. If user needs to address tensor 'operation name', they'll need to specify 'op_name:0' and it will be handles by code below.
I believe that `node = node.get_output_port(node_name)` will return already found place from `input_model.get_place_by_tensor_name(node_name)`, so this code is not necessary

Reply:
Those models work fine on legacy onnx